### DISCUSSION TITLE
Make dependabot less noizy

### DISCUSSION THREAD

**Discussion by abitrolly** [community/community#6078] [Code Security] [answered]
About half of all PRs in https://github.com/pypa/warehouse are created by @dependabot. The majority could be avoided if @dependabot could update existing PRs instead of closing old and opening new when another version of dependency is released. See https://github.com/pypa/warehouse/pull/10148 for example.

I could not find an option to control that behavior. Is that currently possible?

**Comment by jhutchings1** [comment, marked as answer]
Thanks for your feedback! What you described is not currently supported in Dependabot, but it's good feedback! This is also being tracked by https://github.com/dependabot/dependabot-core/issues/3993 in our Dependabot core repository.
